### PR TITLE
Apache and PHP Docker

### PR DESCRIPTION
**Pull request by hmc-berk**

### CODE REVIEW

**Comment by hmc-berk** [comment]
Closes issue 19.

File: app/my-app/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+version: "3.2"
```
Comment (zachChilders) [inline app/my-app/docker-compose.yml]: I don't think we can use this docker-compose directly - we're generating this in the kubernetes layer.

**Review by zachChilders — APPROVED** [review]

File: app/my-app/docker-compose.yml (hunk 1 above)
Comment (zachChilders) [reply app/my-app/docker-compose.yml]: Please remove this file before merging

**Comment by zachChilders** [comment]
I think you have to use a # to close.

Closes #19

**Comment by hmc-berk** [comment]
Closes #19

**Pull request merged**